homeassistant/components/vicare/types.py

```diff
@@ -0,0 +1,21 @@
+"""ViCare types."""
+from enum import StrEnum
+
+
+# Heating programs
+class HeatingProgram(StrEnum):
+    """ViCare preset heating programs.
+
+    As listed in https://github.com/somm15/PyViCare/blob/8ba411483a865e074d1146fd1b8b7a8c4f4be122/PyViCare/PyViCareHeatingDevice.py#L564C27-L564C27
+    """
+
+    ACTIVE = "active"
+    COMFORT = "comfort"
+    ECO = "eco"
+    EXTERNAL = "external"
+    FIXED = "fixed"
+    FORCED = "forcedLastFromSchedule"
+    HOLIDAY = "holiday"
+    NORMAL = "normal"
+    REDUCED = "reduced"
+    STANDBY = "standby"
```

Comment:
probably for heatpumps we need normalheating, normalcooling, ...

Reply:
Yes, but this needs to be implemented in the library first.

Reply:
PR to add more programs in PyViCare https://github.com/somm15/PyViCare/pull/339

Reply:
Will update the sensors once a new version is available 👍